tox.ini

```diff
@@ -1,56 +1,44 @@
 [tox]
-minversion=2.9.0
+minversion = 2.9.0
 envlist = py{27,36},lint,docs
 skipsdist = True
 
 [travis]
-python =
-    2.7: py27
-    3.6: py36
+python = 2.7: py27
+         3.6: py36
 
 [testenv]
 description = run test suite for the application with {basepython}
-setenv =
-    PYTHONPATH={toxinidir}/readthedocs:{toxinidir}
-    DJANGO_SETTINGS_MODULE=readthedocs.settings.test
-    LANG=C
-    DJANGO_SETTINGS_SKIP_LOCAL=True
+setenv = PYTHONPATH={toxinidir}/readthedocs:{toxinidir}
+         DJANGO_SETTINGS_MODULE=readthedocs.settings.test
+         LANG=C
+         DJANGO_SETTINGS_SKIP_LOCAL=True
 deps = -r{toxinidir}/requirements/testing.txt
 changedir = {toxinidir}/readthedocs
-commands =
-    py.test {posargs}
+commands = pytest {posargs}
 
 [testenv:docs]
 description = build readthedocs documentation
 changedir = {toxinidir}/docs
-commands =
-    sphinx-build -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html
+passenv = HOME http_proxy https_proxy no_proxy
+commands = sphinx-build -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
```

Comment:
By arbitrary, I kind of think this is what Anthony means, maybe? (Always start the commands on a new line)